app/views/complaints/_tta_activity_report_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="ct-section-divider">
+  <p class="display-flex flex-justify">
+    <label for="tta-report-id">Activity Report ID</label>
+    <span><%= link_to "Cancel", "#", id: "js-close-tta-form" %></span>
+    <div>
+      <%= form_with url: "/issue_tta_reports", method: "post", class: "usa-search display-flex" do %>
```

Comment:
[suggestion] Probably because of my background at a place where we deployed quite frequently to sub-uris, a url like `/issue_tta_reports` makes me a worried. Could this instead be `url: new_issue_tta_reports` instead?